client/.env

```diff
@@ -1,3 +1,3 @@
-REACT_APP_API_URL=https://api.spoonacular.com/recipes
+REACT_APP_API_URL=spoonacular-recipe-food-nutrition-v1.p.rapidapi.com
 
 REACT_APP_API_KEY=b1ffaa2fe069407e91a03674c1d497ed
```

Comment:
The API key is not updated

Reply:
I didn’t update the key on purpose, keeping it private (Sent it in our chat though)